CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### Unreleased
 
+# v13.14.2
+- Added successfulJobsHistoryLimit and failedJobsHistoryLimit fields for Cronjob resource defined in kubetools.yaml file
```

Comment:
We should reword this now
"Collect additional parameters provided to kubetools cronjob spec and  attach to k8s cronjob spec"

Reply:
https://github.com/EDITD/kubetools/pull/166/commits/7e6bace79a1d21525c5f48dfafd38e24ac3fec6d